src/main/asciidoc/entity-callbacks.adoc

```diff
@@ -0,0 +1,163 @@
+[entity-callbacks]
+= Entity Callbacks
+
+The Spring Data infrastructure provides hooks for modifying an entity before and/or after certain methods are invoked.
```

Comment:
"and/or" can be "and" here. Indicating that both are possible doesn't mean that both are required, which makes the "or" unnecessary. ("and/or" is hardly ever needed.)